feature-libs/cart/saved-cart/styles/_saved-cart-list.scss

```diff
@@ -128,6 +128,7 @@ cx-saved-cart-list {
 
     .cx-saved-cart-list-label {
       color: var(--cx-color-secondary);
+      text-align: start;
```

Comment:
Because we are adding a new style to an existing class, we need to account for breaking changes (ie. some code a user may be using that may change unexpectedly in an upgrade. In this example, they may be using the `cx-saved-cart-list-label` class). 

To account for styling breaking changes, we use `@include forVersion(/* next Spartacus version eg. 6.8 */)` and wrap new changes in this block. This allows customers to be able to "opt-in" to new styling changes by changing their style version from their current version up to the next version (eg. changing their version UP to 6.8 would enable 6.8 version styles.

[Here is an example of using this block](https://github.com/SAP/spartacus/pull/18382/files)